motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -62,7 +63,12 @@ JointTrajectoryAction::JointTrajectoryAction() :
   // A parameter named topics_list contains the respective information for each of the groups
 
   std::string value;
-  ros::param::search("topics_list", value);
+  if(!ros::param::search("topics_list", value))
+  {
+
+    ROS_ERROR_STREAM(industrial_robot_client::motoman_utils::TOPIC_LIST_ERROR_MSG);
+    return;
```

Comment:
Do you want to `return` or actually fail the ctor? If the latter, I think throwing an exception would be more appropriate.